code/modules/mapfluff/ruins/spaceruin_code/TheDerelict.dm

```diff
@@ -175,3 +175,98 @@
 	icon = 'icons/obj/turrets.dmi'
 	icon_state = "turretCover"
 	density = TRUE
+
+/// Captain's log
+/// Credits to goonstation13 for orginal SS13 lore
+/// https://wiki.ss13.co/Storyline and https://www.youtube.com/watch?v=7M-JPH5SOmI
```

Comment:
```suggestion
/// https://wiki.ss13.co/Storyline and https://www.youtube.com/watch?v=FHH1vfY6HTA
```
the better improved version

Reply:
Applied with f69a07cee558338cfab252407ee0d144ffe3a704